File: MiniDOMWebCtrls/DMD/CQSPD/CategorieProdottoModuleHandler.cs

namespace minidom.Forms
{
    public class CategorieProdottoModuleHandler : CBaseModuleHandler
    {
        public CategorieProdottoModuleHandler()
        {
        }

        public override Databases.DBObjectCursorBase CreateCursor()
        {
            var cursor = new Finanziaria.CCategorieProdottoCursor();
            return cursor;
        }
    }
}

File: MiniDOMWebCtrls/DMD/CQSPD/CategorieProdottoModuleHandler.vb
Imports minidom
Imports minidom.Forms
Imports minidom.Databases
Imports minidom.WebSite

Imports minidom.Finanziaria
Imports minidom.Sistema
Imports minidom.Anagrafica

Namespace Forms



    Public Class CategorieProdottoModuleHandler
        Inherits CBaseModuleHandler

        Public Sub New()
            
        End Sub

        Public Overrides Function CreateCursor() As DBObjectCursorBase
            Dim cursor As New CCategorieProdottoCursor
            Return cursor
        End Function


    End Class




End Namespace